docs_src/getting-started/tools/Ch-CommandLineInterface.md

```diff
@@ -1,104 +1,714 @@
+  event            Add, remove and list events
+  help             Help about any command
+  interval         Add, get and list intervals [Support Scheduler]
+  intervalaction   Get, list, update and remove interval actions [Support Scheduler]
+  metrics          Output the CPU/memory usage stats for all EdgeX core/support microservices
+  notification     Add, remove and list notifications [Support Notifications]
+  ping             Ping (health check) all EdgeX core/support microservices
+  provisionwatcher Add, remove, get, list and modify provison watchers [Core Metadata]
+  reading          Count and list readings
+  subscription     Add, remove and list subscriptions [Support Notificationss]
+  transmission     Remove and list transmissions [Support Notifications]
+  version          Output the current version of EdgeX CLI and EdgeX microservices
+
+Flags:
+  -h, --help   help for edgex-cli
+
+Use "edgex-cli [command] --help" for more information about a command.
+```
+
+
+
+## Commands implemented by all microservices
+
+The `ping`, `config`, `metrics` and `version` work with more than one microservice. 
+
+By default these commands will return values from all core and support services:
+
+```
+$ edgex-cli metrics
+Service               CpuBusyAvg MemAlloc MemFrees MemLiveObjects MemMallocs MemSys   MemTotalAlloc
+core-metadata         13         1878936  38262    9445           47707      75318280 5967608
+core-data             13         1716256  40200    8997           49197      75580424 5949504
+core-command          13         1737288  31367    8582           39949      75318280 5380584
+support-scheduler     10         2612296  20754    20224          40978      74728456 4146800
+support-notifications 10         2714480  21199    20678          41877      74728456 4258640
+```
+
+To only return information for one service, specify the service to use:
+
+```
+  -c, --command         use core-command service endpoint
+  -d, --data            use core-data service endpoint
+  -m, --metadata        use core-metadata service endpoint
+  -n, --notifications   use support-notifications service endpoint
+  -s, --scheduler       use support-scheduler service endpoint
+```
+
+Example:
+```
+$ edgex-cli metrics -d
+Service   CpuBusyAvg MemAlloc MemFrees MemLiveObjects MemMallocs MemSys   MemTotalAlloc
+core-data 14         1917712  870037   12258          882295     75580424 64148880
+
+$ edgex-cli metrics -c
+Service      CpuBusyAvg MemAlloc MemFrees MemLiveObjects MemMallocs MemSys   MemTotalAlloc
+core-command 13         1618424  90890    8328           99218      75580424 22779448
+
+$ edgex-cli metrics --metadata
+Service       CpuBusyAvg MemAlloc MemFrees MemLiveObjects MemMallocs MemSys   MemTotalAlloc
+core-metadata 12         1704256  39606    8870           48476      75318280 6139912
+```
+
+The `-j/--json` flag can be used with most of `edgex-go` commands to return the JSON output:
+
+```
+edgex-cli metrics --metadata --json
```

Comment:
I suggest adding a `$ ` here to be aligned with other commands

Reply:
or remove $ from other commands

Reply:
Thanks. I added "$". I have followed the convention of using "$" when we include the output of a command. If the output is not included, then I've not included the "$".